resources/templates/swap.html

```diff
@@ -1,54 +1,31 @@
-{% extends "base.html" %} {% block content %}
-<div class="hero">
-  Get {{ swap_amount }} via a decentralized exchange<br />
-  Choose an exchange of your choice to swap
-</div>
+{% extends "base.html" %}
 
-<div class="action">
-  <div class="menu-item">
+{% block content %}
+  <div class="hero">
+    Get {{ swap_amount }} via a decentralized exchange<br />
```

Comment:
Is this `swap_amount` already formatted with the currency, or would `{{ swap_amount.formatted }}` be better?

Reply:
There is a `__repr__` magic method implemented on the class, that's why it is working. But using the `formatted` attribute would definitely be better. I am changing it.